vhdbuilder/packer/init-variables.sh

```diff
@@ -157,6 +157,8 @@ if [[ "${MODE}" == "linuxVhdMode" ]]; then
 		elif [[ "${IMG_OFFER,,}" == "azure-linux-3" ]]; then
 			# for Azure Linux 3.0, only use AzureLinux prefix
 			SIG_IMAGE_NAME="AzureLinux${SIG_IMAGE_NAME}"
+		elif [[ ${IMG_SKU} == "20_04-lts-cvm" ]]; then
+			SIG_IMAGE_NAME+="TESTDEF"
```

Comment:
don't we already add CVM to the name of the corresponding image definition?

Reply:
This is one of the changes that will need to be made once the PR is ready for merge. We have to upload the image to a specialized image definition. Was hoping to discuss during standup/parking lot. Adding TESTDEF right now is just how I'm getting it there, given that SIG_IMAGE_NAME, which is set to SKU_NAME, points to the definition that we currently have.